packages/suite-desktop/src-electron/modules/auto-updater.ts

```diff
@@ -44,13 +49,27 @@ const init = ({ mainWindow, store }: Dependencies) => {
     let isManualCheck = false;
     let latestVersion: Partial<UpdateInfo> = {};
     let updateCancellationToken: CancellationToken;
+    let shouldInstallUpdateOnQuit = false;
+
     const updateSettings = store.getUpdateSettings();
 
+    // update is downloaded on click and applied automatically whenever user restarts application (now or later)
     autoUpdater.autoDownload = false;
```

Comment:
The combination of the comment and the line beneath it is confusing for me. How does `autoDownload = false` affect `[is] applied automatically whenever user restarts application (now or later)`? Also, reading two lines further, it seems in direct contradiction with `autoUpdater.autoInstallOnAppQuit = false;`.

Please, correct me if I'm wrong, but maybe this comment fits better? (I used an imperative to denote that I'm describing an intention.)
```suggestion
    // Prevent downloading an update unless user explicitly asks for it.
    autoUpdater.autoDownload = false;
```